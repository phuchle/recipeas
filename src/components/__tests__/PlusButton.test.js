import React from 'react';
import PlusButton from '../PlusButton';
import renderer from 'react-test-renderer';

describe('PlusButton', () => {
  test('It renders without crashing', () => {
    const component = renderer.create(<PlusButton />);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});