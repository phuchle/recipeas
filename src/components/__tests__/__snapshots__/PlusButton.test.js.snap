// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlusButton It renders without crashing 1`] = `
<button
  className="undefined glyphicon glyphicon-plus btn btn-primary"
  disabled={false}
  style={
    Object {
      "marginBottom": "10px",
      "marginLeft": "5px",
    }
  }
  type="button"
/>
`;
